Style Guide Consistency Updates (#4420)
mattermost/docs: source/cloud/cloud-integrations/cloud-slash-commands.rst

.. note::

>   This is the admin documentation for slash commands. If you're a developer looking to build an integration, see `our developer documentation <https://developers.mattermost.com/integrate>`__.

Reviewer: This links is "broken" (the page is empty).

Author: Thanks, @amyblais. Fixed via https://github.com/mattermost/docs/pull/4426

You can follow these general guidelines to set up a custom Mattermost slash command for your application.

1. Open **Main Menu > Integrations > Slash Commands**. If you don't have the **Integrations** option in your Main Menu, slash commands may not be enabled or may be disabled for non-admins. Enable them from **System Console > Integrations > Integration Management** or ask your Mattermost System Admin to do so.

> 2. Select **Add Slash Command**, then add name and description for the command.

Reviewer 2: ```suggestion
2. Select **Add Slash Command**, then add a name and description for the command.
```

1. Slash commands are designed to easily allow you to post messages. For other actions such as channel creation, you must also use the `Mattermost APIs <https://api.mattermost.com>`__.
2. You can restrict who can create slash commands in `System Console > Integrations > Integration Management <https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins>`__.
3. Mattermost outgoing webhooks are Slack-compatible. You can copy-and-paste code used for a Slack outgoing webhook to create Mattermost integrations. Mattermost `automatically translates Slack's JSON format <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html#translate-slack-s-data-format-to-mattermost>`__.
4. The external application may be written in any programming language. It needs to provide a URL which receives the request sent by your Mattermost workspace and responds with in the required JSON format.

Share Your Integration
----------------------

> If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://about.mattermost.com/default-app-directory/>`__.

Reviewer 2: ```suggestion
If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://about.mattermost.com/default-app-directory>`__.
```

> The `app directory <https://about.mattermost.com/default-app-directory/>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or on-prem infrastructure.

Reviewer 2: ```suggestion
The `app directory <https://about.mattermost.com/default-app-directory>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or self-managed infrastructure.
```

Troubleshooting
---------------

> See `developer documentation <https://developers.mattermost.com/integrate/slash-commands>`__ for troubleshooting, or `join the Mattermost user community for help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.

Reviewer 2: Should the second URL have a trailing slash?

Author: It appears to resolve with and without the trailing slash.
